dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/export/event/EventRequestParamsMapper.java

```diff
@@ -139,8 +139,14 @@ public EventOperationParams map(EventRequestParams eventRequestParams)
             .occurredBefore(eventRequestParams.getOccurredBefore())
             .scheduledAfter(eventRequestParams.getScheduledAfter())
             .scheduledBefore(eventRequestParams.getScheduledBefore())
-            .updatedAfter(eventRequestParams.getUpdatedAfter())
-            .updatedBefore(eventRequestParams.getUpdatedBefore())
+            .updatedAfter(
+                eventRequestParams.getUpdatedAfter() != null
+                    ? eventRequestParams.getUpdatedAfter().getDate()
+                    : null)
```

Comment:
This type of null unpacking will occur a lot so we should support it with a static helper function to improve readability . Also I wonder if the should have a `toDate()` method instead of `getDate()` to make it more transparent if the `Date` is actually a field called `date` or if we store the date internally in a different way. We might want to go `java.time` API except where we interface with code that still is on `Date`.

Reply:
It would be nice to have everything on `java.time`.
For now I am relying on `DateUtils` methods and most of them are using `java.util.Date`.
I will check in next PR how much work is to use `java.time` in here.